docs/getting-started.md

```
@@ -22,20 +33,16 @@ Afterwards you can interact with solar in the created kind cluster using `kubect
-### Kustomize
-
-To quickly install SolAr on your own kubernetes cluster you can use kustomize:
+### Helm
 
-You will need to ensure [cert-manager](https://cert-manager.io/docs/installation) is installed in the cluster.
+To quickly install SolAr on your own Kubernetes cluster you can use Helm:
 
-First, specify the version you want to install in an environment variable.
+You will need to ensure [cert-manager](https://cert-manager.io/docs/installation) and [Flux](https://fluxcd.io/flux/installation/) (`source-controller` and `helm-controller`) are installed in the cluster — Flux is what reconciles the `OCIRepository`/`HelmRelease` resources SolAr renders, so releases won't roll out without it. If your registries use a private CA, also install [trust-manager](https://cert-manager.io/docs/trust/trust-manager/) and set `caBundle.enabled=true`.
 
 ```shell
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)